### PR TITLE
Adding a new icon_margin attribute the margin in the arrow icon

### PR DESCRIPTION
**Pull request by maruiz81**
Currently, the margin around the Arrow and tic icon is fixed to 16sp. I have added a new parameter called `icon_margin` to let the user change this.

It is especially useful when you try to set the `slider_heigh` smaller because at some point the icon disappears.

I have also added it in the example app. Please, let me know if you miss something

### CODE REVIEW

File: example/src/main/res/layout/content_icon_margin.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <com.ncorti.slidetoact.SlideToActView
+            style="@style/SlideToActView.Example"
+            app:icon_margin="8dp"
+            app:text="Icon margin 8dp" />
+
+        <com.ncorti.slidetoact.SlideToActView
+            style="@style/SlideToActView.Example"
+            app:icon_margin="12dp"
+            app:text="Icon margin 12dp" />
+
```
Comment (cortinico) [inline example/src/main/res/layout/content_icon_margin.xml]: Please add a `"Icon margin 16dp (default)"` example here so users can relate the differences.

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 2]
```diff
@@ -421,21 +426,21 @@ class SlideToActView @JvmOverloads constructor (
...
-            mInnerRect.top.toInt() + mArrowMargin,
-            mInnerRect.right.toInt() - mArrowMargin,
-            mInnerRect.bottom.toInt() - mArrowMargin)
+                mInnerRect.top.toInt() + mArrowMargin,
+                mInnerRect.right.toInt() - mArrowMargin,
+                mInnerRect.bottom.toInt() - mArrowMargin)
         if (mDrawableArrow.bounds.left <= mDrawableArrow.bounds.right &&
-            mDrawableArrow.bounds.top <= mDrawableArrow.bounds.bottom) {
+                mDrawableArrow.bounds.top <= mDrawableArrow.bounds.bottom) {
             mDrawableArrow.draw(canvas)
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: _Commenting here but this is valid for the entire file._

Please cleanup the Diff and make sure you include only changes that are relevant to your change (`mIconMargin` and related fields). Looks like you have a lot of formatting changes. I kindly ask you to remove those. I'll setup ktlint for this purpose in a separate PR.

File: example/src/main/res/layout/content_icon_margin.xml (hunk 1 above)
Comment (maruiz81) [reply example/src/main/res/layout/content_icon_margin.xml]: yes, it makes sense.

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt (hunk 2 above)
Comment (maruiz81) [reply slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: Ok, I will remove it. It seems we have different formating rules and I didn't realise.

**Comment by cortinico** [comment]
Hey @maruiz81, I see you're still committing to this branch. Let me know when you're done and I'll review and merge this 👍

**Comment by maruiz81** [comment]
@cortinico I think is ready. Please check it.

**Comment by cortinico** [comment]
Awesome first contribution @maruiz81 🚀

**Pull request merged**